src/components/LoginSignUp/ForgotPassword.tsx

```diff
@@ -15,11 +13,16 @@ import ForgotPasswordForm from "./ForgotPasswordForm";
 import Header from "./Header";
 import LoginSignUpWrapper from "./LoginSignUpWrapper";
 
-const ForgotPassword = ( ): Node => {
+type RenderProps = {
+  // eslint-disable-next-line react/no-unused-prop-types
```

Comment:
Unsure why this lint is firing, so I silenced it. I think it has to do with the atypical design of the `children` prop on `LoginSignUpWrapper`, which could be addressed in the future.